Add SUL masthead and navbars

camillevilla opened the pull request:
Closes #23 

This PR adds:
- dummy navbar for Login / My Account / Feedback
- Stanford Libraries logo masthead that links to bento homepage
- Library services menu
- Containers and a `layout.scss` file
- View tests for the stacked header

## Current Layout
### XL width
![results page at XL width](https://user-images.githubusercontent.com/5402927/30402731-a4e47bf4-9893-11e7-8361-685125efbaec.png)

### XS width
![results page at XS width](https://user-images.githubusercontent.com/5402927/30402739-ad6938e6-9893-11e7-94d6-2403e5028b73.png)

camillevilla commented:
@jkeck @jvine do users actually need to be able to access login / my account for Bento? I have some dummy links up there for now.

camillevilla commented:
Ok, per our discussion in Slack, the Login link has been removed.

jkeck commented on app/assets/stylesheets/layout.scss:
```diff
@@ -0,0 +1,37 @@
+	#subnavbar, #sul-footer, #masthead {
+    width: 1170px;
+  }
+}
```
It seems like there is some odd alignment going on in this file.

jkeck commented on app/assets/stylesheets/masthead.scss:
```diff
@@ -0,0 +1,23 @@
+#masthead-container {
+  background-color: #9D2520;
```
Wondering if we should create our own variables file to store colors like this (if not already available to us).

camillevilla replied:
@jvine can you clarify which red I should use for the Bento app? 
Mockups have #9D2520, but that didn't map to any brand colors or pre-existing colors in SeachWorks. Should I be using Cardinal Red or Bright Red? 

<img width="842" alt="stanford_reds" src="https://user-images.githubusercontent.com/5402927/30445179-1710864a-993a-11e7-9b96-7ae5ef22377c.png">

camillevilla replied:
Let me know about the following areas: 
(1) font color for the Search bar
(2) background color for the masthead
(3) background color for bento section border-tops

jvine replied:
@camillevilla Grrr omnigraffle doesn't seem to export colours accurately. The red is Cardinal throughout. Everywhere Cardinal. "Search" heading, border-tops, and the logo banner are Cardinal.

"Masthead" has a specific meaning in SW Catalog that I think is different here. 
- page background is #fbfbf9
- logo banner is #8c1515
- subnav menu bar is #797268

Does that cover it?

camillevilla replied:
Yes! Thank you~

jkeck commented on app/assets/stylesheets/masthead.scss:
```diff
@@ -0,0 +1,23 @@
+  #masthead {
+    margin: 0 auto;
+    height: 100px;
+    color: white;
```
We should prob. use the `$white` variable here.

jkeck commented on app/views/shared/_header.html.erb:
```diff
@@ -0,0 +1,27 @@
+  <div id="top-navbar">
+    <ul>
+      <li><a href="https://library.stanford.edu/myaccount">My Account</a></li>
+      <li><a href='#'>Feedback</a></li>
```
@jvine is there somewhere we should send this link in the interim?

jvine replied:
@jkeck you mean with no feedback form hooked up? there's always this:
http://library.stanford.edu/ask/email/feedback
but probably moot by now.

jkeck commented:
In addition to the above comments, I noticed a little issue w/ the link (feedback, library services) alignment.  I think we may need some containers...but not entirely sure.  Totally fine to push that into a separate issue though.

<img width="1020" alt="link-alignment" src="https://user-images.githubusercontent.com/96776/30410584-e25b7b72-98c0-11e7-9b3a-ff936db338c5.png">

camillevilla commented:
Alrighty @jkeck , requested changes have been made:
- moved colors to SCSS variables 
- confirmed bento colors with @jvine
- fixed indent issues in `layout.scss`
- added container to topnav links
